code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -386,6 +385,8 @@
 	grant_abilities(C)
 	return
 
+/datum/species/proc/handle_post_login() //Handles anything not already covered by basic species assignment that requires a client (probably preferences related).
```

Comment:
I thought about using equip but I figured I'd get dusted for it. I'll make the switch